doc/contributing.md

```diff
@@ -0,0 +1,68 @@
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### How Do I Submit A (Good) Issue Report
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+- **Use a clear and descriptive title** for the issue to identify the problem.
+- **Describe the exact steps which reproduce the problem** in as many details as possible. When listing steps, **don't just say what you did, but explain how you did it**.
+- **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+- **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+- **Explain which behavior you expected to see instead and why.**
+- **Which version of cardano-node & cardano-node-tests are you using?**
+
+
+### Your First Code Contribution
+
+#### Pull Requests
+
+The process described here has several goals:
+
+- Maintain cardano-node-tests' quality
+- Fix problems that are important to users
+- Engage the community in working toward the best possible Cardano System Tests
+- Enable a sustainable system for cardano-node-tests' maintainers to review contributions
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
+2. Follow the [styleguides](#styleguides)
+3. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing.
+   If a status check is failing, and you believe that the failure is unrelated to your change,
+   please leave a comment on the pull request explaining why you believe the failure is unrelated.
+   A maintainer will re-run the status check for you. If we conclude that the failure was a false
+   positive, then we will open an issue to track that problem with our status check suite.
+
+While the prerequisites above must be satisfied prior to having your pull request reviewed,
+the reviewer(s) may ask you to complete additional design work, tests, or other changes before
+your pull request can be ultimately accepted.
+
+## Styleguides
+
+### Git Commit Messages
+
+- Use the present tense ("Add feature" not "Added feature")
+- Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+- Limit the first line to 72 characters or less
+- Reference issues and pull requests liberally after the first line
+
+### Python Styleguide
+
```

Comment:
I would keep this information that we have on README.md:

> Run pre-commit install to set up the git hook scripts that will check you changes before every commit. Alternatively run make lint manually before pushing your changes.

It will prevent the failure of the checks at the PR level.

Reply:
done